pkg/geo/wkt/wkt.y

```diff
@@ -41,14 +41,18 @@ type geomPair struct {
 	ends       []int
 }
 
+func makeGeomPair(flatCoords []float64) geomPair {
```

Comment:
let's call `geomPair` something else. maybe `geomFlatCoordsRepr` or something. the pair terminology is getting to me.